### DISCUSSION TITLE
Welcome to html5lib-python Discussions!

### DISCUSSION THREAD

**Discussion by ambv** [html5lib/html5lib-python#560] [Announcements]
## 👋 Welcome!

This is a place to ask questions and engage with other community members. In ancient times we used to have a mailing list, you can still [browse the archives](https://www.mail-archive.com/html5lib-discuss@googlegroups.com/).

Before asking a question here, [check out our friendly docs](https://html5lib.readthedocs.io/en/latest/).

**Comment by willkg** [comment]
It looks like you're taking over maintenance of html5lib. Is that correct? If so, what's the short-term plan?

**Reply by ambv** [reply]
The only plan I have is to stop the bleeding and avoid the domino effect like we've seen with Bleach. The project needs CI in good shape and a new minor release. Other than that, I wouldn't say I'm "taking over". I mean, I was listed as a maintainer here since 2013. It's time to do some maintaining 🤷🏻‍♂️

Sam is still interested in the project. I hope they will be able to in 2023 🤞🏻

**Reply by willkg** [reply]
I appreciate you working on it! Bleach vendors html5lib, but I can update the vendoring when another html5lib release happens. If you need help with things, feel free to tag me in.

Part of my problem with maintaining things is it's so unbelievably lonely. If you put together a slack or a discord or something, I'm game for joining and helping out. I did the 1.0.0 release, but it was a while ago and I don't know if I have maintainer bits anymore.

**Reply by ambv** [reply]
You still do have write access here. Feel free to pick up anything that you'd need happen for Bleach. I avoid IM platforms now because they are too distracting but if you ping me on any Python Discord channel, I'm there.

**Reply by ambv** [reply]
@willkg there's `#html5lib:libera.chat` now. That's IRC so if you're (still) using that you're set. If not, then with [Element](https://element.io/) it feels like just another modern chat app.

**Reply by sethmlarson** [reply]
Super happy to see this happening, thanks @ambv! :clap: I wanted to note that I forwarded the situation+importance of these libraries to Tidelift folks and they currently have $250/month allocated for maintainers of [html5lib](https://tidelift.com/lifter/search/pypi/html5lib) and [bleach](https://tidelift.com/lifter/search/pypi/bleach).

**Reply by Icecracker012** [reply]
I would like to join to this it's awesome
